fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts

```diff
@@ -1393,12 +1523,18 @@ export class FwsCore implements Instrument {
       SimVar.SetSimVarValue('L:A32NX_MASTER_WARNING', 'bool', this.startupCompleted.get() ? warning : false);
     }, true);
 
+    this.fwsAuralVolume.sub((volume) => {
+      // Inhibit master warning/cautions until FWC startup has been completed
+      SimVar.SetSimVarValue('L:A32NX_FWS_AUDIO_VOLUME', 'number', volume);
+    }, true);
+
     this.startupCompleted.sub((completed) => {
       if (completed) {
         // Check if warnings or cautions have to be set
         SimVar.SetSimVarValue('L:A32NX_MASTER_CAUTION', 'bool', this.masterCaution.get());
         SimVar.SetSimVarValue('L:A32NX_MASTER_WARNING', 'bool', this.masterWarningOutput.get());
         SimVar.SetSimVarValue('L:A32NX_FWC_CRC', 'bool', this.auralCrcOutput.get());
+        SimVar.SetSimVarValue('L:A32NX_FWC_CAVALRY_CHARGE', 'bool', this.auralCavalryChargeOutput.get());
```

Comment:
```suggestion
        SimVar.SetSimVarValue('L:A32NX_FWC_CAVALRY_CHARGE', SimVarValueType.Bool, this.auralCavalryChargeOutput.get());
```
In general it's not a good idea to write a var from more than one place. You're better to handle all logic in the subscribable that drives the local var.

Reply:
Moved to MappedSubject which checks the startup condition